--- nix-0.16.1/Cargo.toml	2019-12-24T03:39:07+00:00
+++ nix-0.16.1/Cargo.toml	2020-01-08T23:18:44.181140+00:00
@@ -65,7 +65,3 @@
 version = "3.0.5"
 [target."cfg(any(target_os = \"android\", target_os = \"linux\"))".dev-dependencies.caps]
 version = "0.3.1"
-[target."cfg(target_os = \"dragonfly\")".build-dependencies.cc]
-version = "1"
-[target."cfg(target_os = \"freebsd\")".dev-dependencies.sysctl]
-version = "0.1"
